src/dbtree/root.cpp

```diff
@@ -1374,11 +1367,8 @@ std::string Root::is_board_moved( const std::string& url,
 
         if( is_2ch( url ) ){
 
-            std::list< BoardBase* >::iterator it;
-            for( it = m_list_board.begin(); it != m_list_board.end(); ++it ){
-
+            for( auto& board : m_list_board ) {
                 // 板の最新のrootとpathを取得する
-                BoardBase* board = *( it );
 
                 // 板パスを見て一致したら移転したと見なす
                 // TODO : 板パスが同じ板が2つ以上あるときどうするか？
```

Comment:
cppcheckでSTLのアルゴリズム関数が使えるとレポートがありました。
大きなループ文を書き換えるため別のpull requestで修正したいと思います。

```
src/dbtree/root.cpp:1375:116: style: Consider using std::find_if algorithm instead of a raw loop. [useStlAlgorithm]
                if( is_2ch( board->get_root() ) && url.find( board->get_path_board() + "/" ) != std::string::npos ){
                                                                                                                   ^
```